src/analyze/analyze-security.h

```diff
@@ -13,5 +14,5 @@ typedef enum AnalyzeSecurityFlags {
         ANALYZE_SECURITY_ONLY_LONG_RUNNING = 1 << 2,
 } AnalyzeSecurityFlags;
 
```

Comment:
It would be better to use a more descriptive name instead of v - maybe policy or so